Documentation/user-guides/storage.md

```diff
@@ -46,17 +67,26 @@ spec:
             storage: 40Gi
 ```
 
-> The full documentation of the `storage` field can be found in the [API documentation](../api.md#monitoring.coreos.com/v1.StorageSpec).
+> The full documentation of the `storage` field can be found in the [API reference]({{< ref "api" >}}).
+
+When creating the Prometheus object, a PersistentVolumeClaim is used for each
+Pod in the StatefulSet, and the storage should automatically be provisioned,
+mounted and used.
 
-When creating the Prometheus object, a PersistentVolumeClaim is used for each Pod in the StatefulSet, and the storage should automatically be provisioned, mounted and used.
+The same approach should work with other cloud providers (GCP, Azure, ...) and
+any Kubernetes storage provider supporting dynamic provisioning.
 
 ## Manual storage provisioning
 
-The Prometheus CRD specification allows you to support arbitrary storage through a PersistentVolumeClaim.
+The Prometheus CRD specification allows you to support arbitrary storage
```

Comment:
Just a question but is this part of the guide really necessary in the scope of the operator docs? Also I was under the impression that using NFS was not recommended practice

Reply:
I agree that this part would probably need a rewrite and use another example than NFS but I didn't have time for it.